src/Sarif.Viewer.VisualStudio/ViewModels/ViewModelLocator.cs

```diff
@@ -64,31 +64,31 @@ private static SarifErrorListItem GetDesignTimeViewModel1()
             viewModel.Locations.Add(new Models.CodeFlowLocationModel()
             {
                 FilePath = @"D:\GitHub\NuGet.Services.Metadata\src\Ng\Catalog2Dnx.cs",
-                Region = new Region(11, 1, 11, 2, 0, 0, snippet: null, message: null),
+                Region = new Region(11, 1, 11, 2, 0, 0, 0, 0, snippet: null, message: null),
             });
 
             viewModel.Locations.Add(new Models.CodeFlowLocationModel()
             {
                 FilePath = @"D:\GitHub\NuGet.Services.Metadata\src\Ng\Catalog2Dnx.cs",
-                Region = new Region(12, 1, 12, 2, 0, 0, snippet: null, message: null),
+                Region = new Region(12, 1, 12, 2,0, 0, 0, 0, snippet: null, message: null),
```

Comment:
Missing blank space before "0", here and below.

Reply:
Fixed